src/azure-cli-core/azure/cli/core/authentication.py

```diff
@@ -25,7 +25,7 @@ def _create_scopes(resource):
         scope = resource + '/.default'
     else:
         scope = resource.rstrip('/') + '/.default'
-    return scope
+    return scope,
```

Comment:
>scope, [](start = 11, length = 6)

typo

Reply:
This is a tricky one.

The `scopes` input parameter of `_get_token(...)` function is a tuple:
https://github.com/Azure/azure-cli/blob/b6c8e436888f93bee8a1e4b45700671e754b8500/src/azure-cli-core/azure/cli/core/authentication.py#L40

If the `scopes` argument has multiple items, the original implementation had a bug here:
`token = self._credential.get_token(scopes)`
Because the tuple is not unpacked, the `_credential.get_token(...)` function received a tuple of tuples, for example, here:
https://github.com/Azure/azure-sdk-for-python/blob/aff33bd40c12b16ec3eeaf9ccb7925f884a8a913/sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py#L161
Therefore, during the token acquiring process, the "scope" was the appended string of the elements in the tuple which is of course wrong, the authentication failed.

So, I changed the line to:
`token = self._credential.get_token(*scopes)`

This solved the original problem but led to a different one. The `_create_scopes` function returned a simple string in which the unpack operation resulted a tuple of characters. This, of course, does not work either.

So, I changed the `_create_scopes` function to return a tuple with one element, the `,` after `scope` does this.

I believe this change is necessary to resolve the original bug (independently from the original purpose of this PR), however, I am happy to either add a short comment why it is done so, or moving the tuple creation out of the `_create_scopes` function to the `_get_token` function.

Do you have any preference here?

Reply:
Yes. I prefer `token = self._credential.get_token(*scopes)`.

Reply:
@jiasli Just to be sure: so, are you happy then to return tuple with line `return scope,`, or would you recommend a modification?

Reply:
`return scope,` is good. Let's just add more comments in case of confusion.

Reply:
I have added a short comment to avoid confusion.